kedro/templates/project/hooks/utils.py

```diff
@@ -121,20 +78,20 @@ def setup_template_add_ons(selected_add_ons_list, requirements_file_path, pyproj
         with open(pyproject_file_path, 'a') as file:
             file.write(test_pyproject_requirements)
 
-    if "3" not in selected_add_ons_list:  # If Logging not selected
+    if "Logging" not in selected_add_ons_list:
         logging_yml_path = current_dir / "conf/logging.yml"
         if logging_yml_path.exists():
             logging_yml_path.unlink()
 
-    if "4" not in selected_add_ons_list:  # If Documentation not selected
+    if "Documentation" not in selected_add_ons_list:
         docs_path = current_dir / "docs"
         if docs_path.exists():
             shutil.rmtree(str(docs_path))
     else:
         with open(pyproject_file_path, 'a') as file:
             file.write(docs_pyproject_requirements)
 
-    if "5" not in selected_add_ons_list:  # If Data Structure not selected
+    if "Data Structure" not in selected_add_ons_list:
```

Comment:
Is this name finalised already? I find this a bit awkward and doesn't match the terminology in our docs? cc @amandakys